ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,70 @@
-﻿namespace ConsoleApp1
+﻿namespace Hometask2_DataTypes
 {
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
+    internal class Program
+    {
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="startDate">The start date</param>
+        /// <param name="endDate">The end day</param>
+        /// <param name="searchDay">The searched day</param>
+        /// <returns>Return the number of days of the week within the mentiond period. Return 0 if the searched day doesn't exist. </returns>
+
+        public static int CountMondays(DateTime startDate, DateTime endDate, string searchDay)
+        {
+            for (int j = 0; j < 7; j++)
```

Comment:
I need to check whether the user entered the correct day of the week, so I decided to go this way. Another option that I come up with is to use int instead of string. E.x.:

public static int CountMondays(DateTime startDate, DateTime endDate, int day)
where 1 is for Monday, 2 for Tuesday and so on

It allows to avoid using a loop in this piece of code, but still I'm not sure if it's the better solution.
By the way, did you mean to ignore loops only in this part or throughout the code since I have a few more loops there?

If my second option is not as good either, pls, give me a lil hint in what way this issue could be solved, just some clue to think over.